src/main.rs

```diff
@@ -134,27 +153,29 @@ fn main() {
                 match status {
                     None => {
                         #[cfg(feature = "debug_ctap")]
-                        print_packet_notice("Sending packet timed out", &timer);
+                        print_packet_notice("Sending packet timed out", &clock);
                         // TODO: reset the ctap_hid state.
                         // Since sending the packet timed out, we cancel this reply.
                         break;
                     }
                     Some(usb_ctap_hid::SendOrRecvStatus::Error) => panic!("Error sending packet"),
                     Some(usb_ctap_hid::SendOrRecvStatus::Sent) => {
                         #[cfg(feature = "debug_ctap")]
-                        print_packet_notice("Sent packet", &timer);
+                        print_packet_notice("Sent packet", &clock);
                     }
                     Some(usb_ctap_hid::SendOrRecvStatus::Received) => {
                         #[cfg(feature = "debug_ctap")]
-                        print_packet_notice("Received an UNEXPECTED packet", &timer);
+                        print_packet_notice("Received an UNEXPECTED packet", &clock);
                         // TODO: handle this unexpected packet.
                     }
                 }
             }
         }
 
-        let now = timer.get_current_clock().flex_unwrap();
-        if let Some(wait_duration) = now.wrapping_sub(last_led_increment) {
+        let now = clock.try_now().unwrap();
+        use core::convert::TryInto;
```

Comment:
I think importing on the top of the file is cleaner.

Reply:
Done.